hawkular-wildfly-monitor/src/main/assembly/module-assembly.xml

```diff
@@ -39,6 +39,14 @@
       <directoryMode>0755</directoryMode>
     </fileSet>
   </fileSets>
+  <files>
+    <file>
+      <source>${project.basedir}/src/main/assembly/subsystem.xml</source>
+      <outputDirectory>/${moduleDir}</outputDirectory>
+      <destName>subsystem-snippet.xml</destName>
+    </file>
+  </files>
+
 
   <dependencySets>
     <dependencySet>
```

Comment:
will this new snippet file cause problems if someone wants to manually install the module by just unzipping in the WildFly modules/ directory? We don't want to FORCE someone to have to use this installer to install our special module .zip file.